modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/PriceTag.java

```diff
@@ -232,6 +232,7 @@ private PriceModel _getPriceModel(
 				CPInstanceUnitOfMeasure cpInstanceUnitOfMeasure =
 					cpInstanceUnitOfMeasures.get(0);
 
+				_quantity = cpInstanceUnitOfMeasure.getIncrementalOrderQuantity();
```

Comment:
shouldn't we take into consideration the input quantity here?
if we have an IncrementalOrderQuantity that is 50, this code is ok when we first see the product with base data.
but if we increase the quantity (by using the quantity selector) say we go up 1 step, our quantity will actually be IncrementalOrderQuantity * quantity selected = 100
so we need the price for 100 in this case not 50
am I missing something?

Reply:
I think you should do something like this https://github.com/liferay/liferay-portal/blob/e9c93faeaa8c3ad54b80f37da4c5d816bb0c9d4f/modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/quantities.js#L11 to find the actual min quantity before the tag starts do what it's supposed to do

Reply:
@luca-pellizzon , I will double check, but I believe this tag will only be hit one time. The input quantity has never affected this tag before I believe. Even if you increase the quantity, the price tag will not be called again.